lib/oath/views/account_view.dart

```diff
@@ -99,6 +99,24 @@ class _AccountViewState extends ConsumerState<AccountView> {
     }).toList();
   }
 
+  String? a11yCredentialLabel(String? issuer, String? name, OathCode? code) {
+    String? label = '';
+    String? tmpIssuer = issuer;
+    String? tmpName = name;
+    String? tmpCode = code?.value;
+    if (tmpIssuer != null) {
+      label += tmpIssuer;
+    }
+    if (tmpName != null) {
+      label += tmpName;
+    }
+    if (tmpCode != null) {
+      label += tmpCode;
+    }
```

Comment:
Don't we need spaces between the parts for this to be read out correctly?

Reply:
Good catch. Absolutely. Since account name is required I will not add a space to it, but instead add a space before issuer and code.